libs/command/account_client.go

```diff
@@ -0,0 +1,22 @@
+package command
+
+import (
+	"context"
+
+	"github.com/databricks/databricks-sdk-go"
+)
+
+func SetAccountClient(ctx context.Context, a *databricks.AccountClient) context.Context {
+	if v := ctx.Value(accountClientKey); v != nil {
+		panic("command.SetAccountClient called twice on the same context")
+	}
+	return context.WithValue(ctx, accountClientKey, a)
```

Comment:
Before move this looked like this:

```
func SetAccountClient(ctx context.Context, a *databricks.AccountClient) context.Context {
	return context.WithValue(ctx, &accountClient, a)
}
```

It did not panic. The PR description does not mention adding panics. If you like to add panic, can you have a separate PR for that?

Move should be pure move, otherwise it's difficult to review.

Reply:
Fair enough, I'll add the panic in a fast followup.